### DISCUSSION TITLE
Empty lines are not supported

### DISCUSSION THREAD

**Discussion by HaujetZhao** [vslinko/obsidian-outliner#166] [Ideas]
**Describe the bug**
write such a list: 

```markdown
- 1

- 2

- 3

  three

  Triple

  - 3.1
  - 3.2

- 4

  - 4.5
```

it is not supoorted by outliner.

**Comment by vslinko** [comment]
@HaujetZhao Thank you for your contribution!

The problem is not with paragraphs, but with empty lines. Empty lines are not supported.
